package/tests/test_integrations/test_sqlite_store.py

```diff
@@ -29,13 +29,166 @@ def db_session_class(store_path):
     return Session
 
 
+@pytest.fixture(scope="class")
+def aws_credentials():
+    """Mocked AWS credentials for moto"""
+    os.environ["AWS_ACCESS_KEY_ID"] = "testing"
+    os.environ["AWS_SECRET_ACCESS_KEY"] = "testing"
+    os.environ["AWS_SESSION_TOKEN"] = "testing"
+
+
+@pytest.fixture(scope="class")
+def mocked_s3_bucket(aws_credentials):
+    """S3 Mock Client"""
+    with mock_s3():
+        conn = boto3.client("s3", region_name="us-east-1")
+        conn.create_bucket(Bucket=BUCKET_NAME)
+        yield conn
+
+
+@pytest.fixture
+def remote_path():
+    return f"s3://{BUCKET_NAME}"
+
+
+@pytest.fixture
+def mock_db1(tmp_path):
+    database_loc = str(tmp_path / "db1.db")
+    engine = create_engine(f"sqlite:///{database_loc}")
+    session_class = sessionmaker(bind=engine)
+    Base.metadata.create_all(bind=engine)
+    session = session_class()
+    session.add(RunModel(id="1", blob="blob1"))
+    session.commit()
+    session.close()
+    yield Path(database_loc)
+
+
+@pytest.fixture
+def mock_db2(tmp_path):
+    database_loc = str(tmp_path / "db2.db")
+    engine = create_engine(f"sqlite:///{database_loc}")
+    session_class = sessionmaker(bind=engine)
+    Base.metadata.create_all(bind=engine)
+    session = session_class()
+    session.add(RunModel(id="2", blob="blob2"))
+    session.commit()
+    session.close()
+    yield Path(database_loc)
+
+
+@pytest.fixture
+def mock_db3(tmp_path):
+    database_loc = str(tmp_path / "db3.db")
+    engine = create_engine(f"sqlite:///{database_loc}")
+    session_class = sessionmaker(bind=engine)
+    Base.metadata.create_all(bind=engine)
+    session = session_class()
+    session.add(RunModel(id="3", blob="blob3"))
+    session.commit()
+    session.close()
+    yield Path(database_loc)
+
+
+@pytest.fixture
+def mock_db4(tmp_path):
+    database_loc = str(tmp_path / "db4.db")
+    engine = create_engine(f"sqlite:///{database_loc}")
+    session_class = sessionmaker(bind=engine)
+    Base.metadata.create_all(bind=engine)
+    session = session_class()
+    session.add(RunModel(id="3", blob="blob3"))
+    session.commit()
+    session.close()
+    yield Path(database_loc)
```

Comment:
That's a lot of very similar looking fixtures. I think it would be helpful to add a comment to each to make it clear what purpose it solves. It took me a while to figure out that db4 has the same data as db3, so if you add something about that being a duplicate it would help understanding the use quicker.

Reply:
And do you actually need 4? Wouldn't 3 be enough? 1 and 2 as unique dbs and then 3 as a duplicate of 2?

Reply:
makes sense. I will change